scenarios/evaluate/evaluate_remotely/evaluate_remotely.ipynb

```diff
@@ -0,0 +1,192 @@
+    "- [Azure AI Safety Evaluation](https://aka.ms/azureaistudiosafetyeval)\n",
+    "- [azure-ai-evaluation](https://learn.microsoft.com/en-us/azure/ai-studio/how-to/develop/evaluate-sdk)\n",
+    "\n",
+    "## Time\n",
+    "\n",
+    "You should expect to spend 20 minutes running this sample. \n",
+    "\n",
+    "## About this example\n",
+    "\n",
+    "This example demonstrates the remote evaluation of query and response pairs that were generated by an LLM model. It is important to have access to AzureOpenAI credentials and an AzureAI project. **To create data to use in your own evaluation, learn more [here](https://learn.microsoft.com/en-us/azure/ai-studio/how-to/develop/simulator-interaction-data)**\n",
+    "\n",
+    "## Before you begin\n",
+    "### Prerequesite\n",
+    "- [Have an online deployment on Azure Open AI studio supporting `chat completion` such as `gpt-4`](https://learn.microsoft.com/en-us/azure/machine-learning/concept-endpoints-online?view=azureml-api-2)\n",
+    "- You also might want to evaluate data generated by your LLM, to see how to generate data to be evaluated using the Azure AI Evaluation SDK, see our samples on simulation \n",
+    "\n",
+    "### Installation\n",
+    "\n",
+    "Install the following packages required to execute this notebook. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip uninstall azure-ai-project azure-ai-ml azure-ai-evaluation\n",
+    "%pip install azure-identity\n",
+    "%pip install https://remoteevalbugbash.blob.core.windows.net/remoteevalbugbash/azure_ai_evaluation-1.0.0a20241022005-py3-none-any.whl\n",
+    "%pip install https://remoteevalbugbash.blob.core.windows.net/remoteevalbugbash/azure_ai_project-1.0.0b1-py3-none-any.whl"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.project import AIProjectClient\n",
+    "from azure.identity import DefaultAzureCredential\n",
+    "from azure.ai.project.models import Evaluation, Dataset, EvaluatorConfiguration, ConnectionType\n",
+    "from azure.ai.evaluation import F1ScoreEvaluator, RelevanceEvaluator, ViolenceEvaluator"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Connect to your Azure Open AI deployment\n",
+    "To evaluate your LLM-generated data remotely in the cloud, we must first connect to your Azure Open AI deployment. This deployment must be a GPT model which supports `chat completion`, such as `gpt-4`. To see the connection string, navigate to the \"Project Overview\" page for your Azure AI project. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "project_client = AIProjectClient.from_connection_string(\n",
+    "    credential=DefaultAzureCredential(),\n",
+    "    conn_str=\"<connection_string>\",\n",
```

Comment:
I think several folks have been struggling with the connection string, can we provide an example in-line commented out?